uberfire-server/src/main/java/org/uberfire/server/BaseUploadServlet.java

```diff
@@ -64,14 +65,21 @@ protected ServletFileUpload getServletFileUpload() {
     protected void writeFile(final IOService ioService,
                              final Path path,
                              final FileItem uploadedItem) throws IOException {
-        if (!ioService.exists(path)) {
-            ioService.createFile(path);
-        }
+        try {
+            ioService.startBatch(path.getFileSystem());
 
-        ioService.write(path,
-                        IOUtils.toByteArray(uploadedItem.getInputStream()));
+            if (!ioService.exists(path)) {
+                ioService.createFile(path);
+            }
 
-        uploadedItem.getInputStream().close();
+            ioService.write(path,
+                            IOUtils.toByteArray(uploadedItem.getInputStream()));
+        } catch (final Exception e) {
+            throw new RuntimeException(e);
```

Comment:
I don't see benefit wrapping any `Exception` to `RuntimeExcetpion`, could you explain please?

Reply:
@jomarko It was copy and paste demons! I've now simply wrapped the ```ioService``` operations in a ```try {..} finally {..}``` block... that is also in keeping with how error handling was before my change; so less likely to introduce any regressions in any existing error handling.